scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml

```diff
@@ -297,6 +297,46 @@ DFN-8-1EP_3x3mm_P0.65mm_EP1.7x2.05mm:
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
 
+DFN-8-1EP_4x4mm_P0.8mm_EP2.3x3.24mm:
+  device_type: 'DFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.st.com/resource/en/datasheet/ld1086.pdf'
+  # ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  body_size_x:
+    minimum: 3.90
+    nominal: 4
+    maximum: 4.10
+  body_size_y:
+    minimum: 3.90
+    nominal: 4
+    maximum: 4.10
+  lead_width:
+    minimum: 0.23
+    nominal: 0.30
+    maximum: 0.38
+  lead_len:
+    minumum: 0.4
+    nominal: 0.5
+    maximum: 0.6
+
+  EP_size_x: 2.30
+  EP_size_y: 3.24
```

Comment:
Enter size of the package ep with this parameter and use EP_size_x_overwrite to force it to use the pad size as defined in the datasheets suggested footprint. (same with y size)

Reply:
So I made the changes but the EP didn't get overridden. Or did I fuck up?

Reply:
You have a typo in the parameter name.